### PR TITLE
add new packages luaosc luapack

### PR DESCRIPTION
**Pull request by lynxis**
Maintainer: me
Compile tested: ar71xx
Run tested: ar71xx
Description:

add new packages:
- luaosc: a simple luAdd opensoundcontrol library
- lualpack: Lua lpack libary to pack and unpack binary value

### CODE REVIEW

**Comment by karlp** [comment]
What does luapack do over say http://www.inf.puc-rio.br/~roberto/struct/  ? (packaged at https://github.com/remakeelectric/owrt_pub_feeds/blob/master/luastruct/Makefile I never upstreamed it as we stopped using it)

**Comment by lynxis** [comment]
@karlp it looks like the same. But I only packaged it, because it's a requirement of the luaosc.

**Comment by lynxis** [comment]
@karlp do you have any reason not to merge this?

**Comment by karlp** [comment]
no no, was just curious what it did differently.  It seems a very obscure piece of code :)

File: lang/lualpack/Makefile [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2015 OpenWrt.org
```
Comment (diizzyy) [inline lang/lualpack/Makefile]: This shouldn't be openwrt.org nowdays?

**Comment by lynxis** [comment]
ping

**Comment by champtar** [comment]
@lynxis 
I prefer when commit start like "lualpack: add new package"
else you can merge yourself i think

**Comment by champtar** [comment]
@lynxis sorry i was confused by the "contributor",
if you can just change the commit message, I'll happily merge your commits

**Comment by diizzyy** [comment]
Hmm... Is this something that's going to be used widely? As karlp pointed out, it's pretty obscure and the Makefiles need work, PKG_MD5SUM should be PKG_HASH. That said, we've turned down others before because we want to avoid "code-rot" because something is really obscure and/or upstream is dead.

**Comment by lynxis** [comment]
i'm closing this for now. I'll make sure, this package is still be used by some people before opening again.

**Pull request closed without merging**